docs/source/distributed.tensor.parallel.rst

```diff
@@ -70,12 +70,7 @@ since we are parallelizing on the head dim of the ``MultiheadAttention`` module.
 
 
 We also support 2D parallelism, where we compose tensor parallelism with data parallelism.
-To integrate with ``FullyShardedDataParallel``,
-users just need to call the following API explicitly:
-
-
-.. currentmodule:: torch.distributed.tensor.parallel.fsdp
-.. autofunction::  enable_2d_with_fsdp
+To integrate with ``FullyShardedDataParallel``.
 
 
 To integrate with ``DistributedDataParallel``,
```

Comment:
any reason we don't remove the DDP extension part yet?

Reply:
The DDP part is not ready yet, but I guess we can also remove it from the doc if we don't want to promote it anyway.